src/main/java/com/feed_the_beast/ftbu/config/FTBUConfigWorld.java

```diff
@@ -30,4 +30,8 @@
 
     @ConfigValue(id = "world.temp.max_loaded_chunks", file = FTBUFinals.MOD_ID, info = "Temp config for max chunks that player can load")
     public static final PropertyShort MAX_LOADED_CHUNKS = new PropertyShort(64, 0, 30000);
+
+    @ConfigValue(id = "world.locked_in_claimed_chunks", file = FTBUFinals.MOD_ID, info = "Player is allowed to interact only in her claimed chunks")
```

Comment:
"Players are allowed to interact only in their claimed chunks" would be better, because its gender neutral